app/assets/scripts/components/home/index.js

```diff
@@ -12,105 +21,369 @@ import {
-                  Use the dashboard to interact with real NASA data and investigate how
-                  social distancing measures and regional shelter-in-place guidelines have
-                  affected Earth’s air, land, and water. Explore individual &apos;Spotlight Areas&apos;
-                  to see how the indicators in each specific location have changed through
-                  time.
-                </p>
-              </Prose>
-              <InpageHGroup
-                title='Explore the site'
+            <Intro>
+              <IntroCopy>
+                <IntroTitle>
+                  <small>Welcome to the</small>
+                  <strong>COVID-19 Data Dashboard</strong>
+                </IntroTitle>
+                <IntroProse>
+                  <p>
+                    Interact with real NASA data and
+                    investigate how social distancing measures and regional
+                    shelter-in-place guidelines have affected Earth’s air, land,
+                    and water. Explore individual &apos;Spotlight Areas&apos; to
+                    see how the indicators in each specific location have
+                    changed through time.
+                  </p>
+                </IntroProse>
+                <IntroStats>
+                  <dt>Locations</dt>
+                  <dd><Link to='/spotlight' title='Explore the spotlight areas'>06</Link></dd>
+                  <dt>Indicators</dt>
+                  <dd><Link to='/indicators' title='Learn about the indicators'>20</Link></dd>
+                  <dt>Stories</dt>
+                  <dd><Link to='' title='Dive into the stories'>04</Link></dd>
+                </IntroStats>
+                <IntroStories>
+                  <IntroStoriesTitle>Did you know?</IntroStoriesTitle>
+                  <Story>
+                    <StoryTitle>{currentStory.title}</StoryTitle>
+                    <StoryProse>
+                      <p>{currentStory.prose}</p>
+                    </StoryProse>
+                    <StoryActions>
+                      <Button
+                        as='a'
+                        title='View the global map'
+                        to='/'
+                        variation='base-raised-light'
+                        useIcon={['chevron-right--small', 'after']}
+                      >
+                        Learn more
+                      </Button>
+                    </StoryActions>
+                  </Story>
+                </IntroStories>
+              </IntroCopy>
+              {/*
+              <IntroMedia>
+              </IntroMedia>
+              */}
+              <MbMap
+                ref={this.mbMapRef}
+                onAction={this.onMapAction}
+                layers={[]}
+                activeLayers={[]}
+                // date={this.state.timelineDate}
+                aoiState={null}
+                comparing={false}
+              />
+              <Previous
+                as='a'
```

Comment:
```suggestion
                element='a'
```
Because `Button` is a custom and complex component is better to use `element`